File: Linguagem C++/Estrutura Condicional/Dardo.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    double d1,d2,d3,maior;

    cout << "Digite as tres distancias:" << endl;
    cin >> d1;
    cin >> d2;
    cin >> d3;

    maior = 0;
    if (d1 > d2 && d1 > d3){
        maior = d1;
    }else if(d2 > d3){
        maior = d2;
    }else{
        maior = d3;
    };
    cout << fixed << setprecision(2);
    cout << "MAIOR DISTANCIA: " << maior << endl;

    return 0;
}

File: Linguagem Java/Exercicios/Dardo.java
import java.util.Locale;
import java.util.Scanner;

public class Dardo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double d1,d2,d3,maior;
		
		System.out.println("Digite as tres distancias: ");
		d1 = sc.nextDouble();
		d2 = sc.nextDouble();
		d3 = sc.nextDouble();
		
		if (d1 > d2 && d1 > d3) {
			maior = d1;
			
		}else if (d2 > d3) {
			maior = d2;			
		}else {
			maior = d3;			
		};
		
		System.out.println("MAIOR DISTANCIA = "+String.format("%.2f", maior));
		
		sc.close();

	}

}
